engine/src/test/java/org/camunda/bpm/engine/test/api/multitenancy/query/MultiTenancyUserOperationLogQueryTest.java

```diff
@@ -0,0 +1,223 @@
+  protected static final String USER_ONE = "aUserId";
+  protected static final String USER_TWO = "aUserId1";
+  protected static final String USER_WITHOUT_TENANT = "aUserId1";
+
+  protected static final String TENANT_ONE = "tenant1";
+  protected static final String TENANT_TWO = "tenant2";
+  protected static final String PROCESS_NAME = "process";
+
+  protected static final BpmnModelInstance MODEL = Bpmn.createExecutableProcess(PROCESS_NAME)
+      .startEvent().userTask("aTaskId").done();
+
+  protected ProcessEngineRule engineRule = new ProvidedProcessEngineRule();
+  protected ProcessEngineTestRule testRule = new ProcessEngineTestRule(engineRule);
+
+  @Rule
+  public RuleChain ruleChain = RuleChain.outerRule(engineRule).around(testRule);
+
+  protected TaskService taskService;
+  protected HistoryService historyService;
+  protected RuntimeService runtimeService;
+  protected IdentityService identityService;
+
+  @Before
+  public void init() {
+    taskService = engineRule.getTaskService();
+    historyService = engineRule.getHistoryService();
+    identityService = engineRule.getIdentityService();
+    runtimeService = engineRule.getRuntimeService();
+  }
+
+  @Test
+  public void shouldReturnNoResultsWithoutTenant() {
+    // given logs with assigned tenant
+    testRule.deployForTenant(TENANT_ONE, MODEL);
+    identityService.setAuthentication(USER_ONE, null, Arrays.asList(TENANT_ONE));
+
+    runtimeService.startProcessInstanceByKey(PROCESS_NAME);
+    String taskId = taskService.createTaskQuery().singleResult().getId();
+
+    taskService.complete(taskId);
+
+    // when authenticated user without tenants
+    identityService.setAuthentication(USER_WITHOUT_TENANT, null);
+    UserOperationLogEntry singleResult = historyService.createUserOperationLogQuery()
+        .entityType(EntityTypes.TASK)
+        .singleResult();
+
+    // then
+    assertThat(singleResult).isEqualTo(null);
+  }
+
+  @Test
+  public void shouldReturnResultsWhenMultipleTenants() {
+    // given task with no tenant, with TENANT_ONE, and with TENANT_TWO
+    testRule.deploy(MODEL);
+    runtimeService.startProcessInstanceByKey(PROCESS_NAME);
+    identityService.setAuthenticatedUserId(USER_WITHOUT_TENANT);
+    String taskId = taskService.createTaskQuery().singleResult().getId();
+    taskService.complete(taskId);
+
+    testRule.deployForTenant(TENANT_ONE, MODEL);
+    identityService.setAuthentication(USER_ONE, null, Arrays.asList(TENANT_ONE));
+    runtimeService.createProcessInstanceByKey(PROCESS_NAME)
+    .processDefinitionTenantId(TENANT_ONE)
+    .execute();
+    taskId = taskService.createTaskQuery().tenantIdIn(TENANT_ONE).singleResult().getId();
+    taskService.complete(taskId);
+
+    testRule.deployForTenant(TENANT_TWO, MODEL);
+    identityService.setAuthentication(USER_TWO, null, Arrays.asList(TENANT_TWO));
+    runtimeService.createProcessInstanceByKey(PROCESS_NAME)
+    .processDefinitionTenantId(TENANT_TWO)
+    .execute();
```

Comment:
Indentation seems to be off here.
```suggestion
        .processDefinitionTenantId(TENANT_TWO)
        .execute();
```